assets/js/hooks/cell.js

```diff
@@ -250,7 +250,11 @@ const Cell = {
 
   maybeFocusCurrentEditor(scroll = false) {
     if (this.isFocused && this.insertMode) {
-      this.currentEditor().focus();
+      const editor = this.currentEditor();
+
+      if (!editor) return;
+
+      editor.focus();
```

Comment:
Is there an easy scenario to reproduce this? Given how we use this function I'd expect the editor to always be in place, so just want to figure out what I'm missing :D

Reply:
The issue that @aleDsz showed to me yesterday is that you can't click on the output of the setup cell.

Reply:
Yeah, but this one is another issue as far as I understand. For the click see the other comment :)

Reply:
> Is there an easy scenario to reproduce this? Given how we use this function I'd expect the editor to always be in place, so just want to figure out what I'm missing :D

This was hard to get because I was testing the other case with setup output, but when I created a new code block to test some js functions, this one started to print error because `focus()` didn't exist in that situation, but when I clicked inside the code editor, it stopped.

I think, for some reason, editor became null/undefined and then updated his value with the current editor.